hack/pre-release.sh

```diff
@@ -227,7 +227,7 @@ github_update() {
 
   # remove make bundle artifacts
   sed -i 's/REPLACE_IMAGE:latest/REPLACE_IMAGE/' bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
-  sed -i 's/operator-sdk-v1.14.0+git/operator-sdk-v1.15.0+git/' bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
+  sed -i 's/operator-sdk-v1.14.0+git/operator-sdk-v1.32.0+git/' bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
```

Comment:
Please use the `OPERATOR_SDK_VERSION` variable here so that we don't need to update this line for future SKD version changes